### PR TITLE
feat(as-4236): link full-planning eligibility journey upto graded-or-refused

### PR DESCRIPTION
**Pull request by nssdesikan**
# Ticket Number

https://pins-ds.atlassian.net/browse/AS-4236

## Description of change
link full-planning eligibility journey up to granted-or-refused

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/e2e-tests/cypress/integration/full-planning/before-you-start/granted-or-refused-application/granted-or-refused-application-steps.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const pageTitle = 'Was your planning application granted or refused? - Before yo
 const pageHeading = 'Was your planning application granted or refused?';
 const url = '/before-you-start/granted-or-refused';
 const decisionDatePageUrl = '/before-you-start/decision-date';
-const decisionDateDuePageUrl = '/before-you-start/decision-date-due';
```
Comment (swdpcomputing) [inline packages/e2e-tests/cypress/integration/full-planning/before-you-start/granted-or-refused-application/granted-or-refused-application-steps.js:16]: Just checking this isn't the decision date page, as that url is different: https://pins-ds.atlassian.net/browse/AS-3673?atlOrigin=eyJpIjoiOWRmODU5NmE3Y2Y0NGU3Njk1YzI5NmJiYWNhY2Q4MjEiLCJwIjoiaiJ9

File: packages/forms-web-app/__tests__/unit/controllers/full-planning/granted-or-refused.test.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ describe('controllers/full-planning/granted-or-refused', () => {
       const pageRedirect =
         grantedOrRefusedPlanningApplicationController.forwardPage('nodecisionreceived');
 
-      expect(pageRedirect).toEqual('/before-you-start/decision-date-due');
```
Comment (swdpcomputing) [inline packages/forms-web-app/__tests__/unit/controllers/full-planning/granted-or-refused.test.js:53]: Just checking this isn't the decision date page, as that url is different: https://pins-ds.atlassian.net/browse/AS-3673?atlOrigin=eyJpIjoiOWRmODU5NmE3Y2Y0NGU3Njk1YzI5NmJiYWNhY2Q4MjEiLCJwIjoiaiJ9

File: packages/forms-web-app/src/controllers/full-planning/granted-or-refused.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const {
 exports.forwardPage = (status) => {
   const statuses = {
     [applicationStatus.GRANTED]: '/before-you-start/decision-date',
-    [applicationStatus.NODECISION]: '/before-you-start/decision-date-due',
+    [applicationStatus.NODECISION]: '/before-you-start/date-decision-due',
```
Comment (swdpcomputing) [inline packages/forms-web-app/src/controllers/full-planning/granted-or-refused.js:17]: Just checking this isn't the decision date page, as that url is different: https://pins-ds.atlassian.net/browse/AS-3673?atlOrigin=eyJpIjoiOWRmODU5NmE3Y2Y0NGU3Njk1YzI5NmJiYWNhY2Q4MjEiLCJwIjoiaiJ9

**Review by swdpcomputing — COMMENTED** [review]
Same question comment in three places.

File: packages/e2e-tests/cypress/integration/full-planning/before-you-start/granted-or-refused-application/granted-or-refused-application-steps.js (hunk 1 above)
Comment (nssdesikan) [reply packages/e2e-tests/cypress/integration/full-planning/before-you-start/granted-or-refused-application/granted-or-refused-application-steps.js:16]: No, this is not _decision date_ page that you worked on

File: packages/forms-web-app/__tests__/unit/controllers/full-planning/granted-or-refused.test.js (hunk 2 above)
Comment (nssdesikan) [reply packages/forms-web-app/__tests__/unit/controllers/full-planning/granted-or-refused.test.js:53]: No, this is not _decision date_ page that you worked on

File: packages/forms-web-app/src/controllers/full-planning/granted-or-refused.js (hunk 3 above)
Comment (nssdesikan) [reply packages/forms-web-app/src/controllers/full-planning/granted-or-refused.js:17]: No, this is not _decision date_ page that you worked on

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**